backup-server/watcher/src/App.js

```diff
@@ -88,7 +88,7 @@ export default class App extends Component {
   }
 
   forceBackup() {
-    fetch(`http://35.211.175.158:5000/forcebackup`, {
+    fetch('https://cors-anywhere.herokuapp.com/http://35.211.175.158:5000/forcebackup', {
```

Comment:
What is this for? are we getting cors errors on the server?

Reply:
ok i'm going to explain this meme style:
> try localhost:5000
> doesn't work
> ![image](https://user-images.githubusercontent.com/26289557/49539602-fe9b2580-f89b-11e8-93ec-c242a280f611.png)

so i'm like ok, maybe i'll put ip
> try http://35.211.175.158:5000
> works FOR LITERALLY EVERY OTHER API
> doesn't work for this api
>![image](https://user-images.githubusercontent.com/26289557/49539602-fe9b2580-f89b-11e8-93ec-c242a280f611.png)

> so i just use work around
> works